packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -285,11 +285,13 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
             isCircular
             onClick={onSortDirectionChange()}
           />
+          <div className='filter'>
+            {filter}
+          </div>
```

Comment:
I'm not sure if we should wrap it into that div. In `filter` it already has a `div.filter-tags` so we should style it there or delete it.

Reply:
✅